### PR TITLE
Dialog and Text Area

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image

**Features**
- Danger Buttons support
  (not yet published in [SK Components](https://github.com/suankularb-wittayalai-school/sk-components), pending review in https://github.com/suankularb-wittayalai-school/sk-components/pull/32)
- Dialog
  - Dialog Header
  - Dialog Section
  - Dialog List
  - Dialog Actions
- Text Area

**Fixes**
- Card Header label is required
- Some elements don’t allow JSX elements as text
- Unneccessary duplicate CSS imports

### CODE REVIEW

File: src/components/Input/Dropdown.tsx [hunk 1]
```diff
@@ -5,9 +5,6 @@ import { useState } from "react";
 // Components
 import MaterialIcon from "../Icon";
 
-// Styles
-import "@suankularb-components/css/dist/css/suankularb-components.min.css";
```
Comment (Jimmy-Tempest) [inline src/components/Input/Dropdown.tsx:9]: I'm not really sure but why is this removed?

**Review by Jimmy-Tempest — APPROVED** [review]
All good but can I have a explaination on the removal of the styles import?

File: src/components/Input/Dropdown.tsx (hunk 1 above)
Comment (SiravitPhokeed) [reply src/components/Input/Dropdown.tsx:9]: It still shows up in the build and I can still use the library with `npm link` so I don’t see why we need to import the styles in every single file. We still have to publish it though to see if it’d fully work but I don’t see why not.

**Review by IHasDiabetes — APPROVED** [review]
LGTM!
